### ISSUE TITLE
Training and validation loss

### ISSUE THREAD

**Issue by hdnhan** [Grace9994/CoMoSVC#12]
I observed that training loss is decreasing in overall, but validation loss is increasing. Have you experienced the same behavior? 

Thanks.

<img width="884" alt="image" src="https://github.com/Grace9994/CoMoSVC/assets/22287261/6eae07dd-06c4-4e53-8d3f-b264c0af3d2b">

<img width="887" alt="image" src="https://github.com/Grace9994/CoMoSVC/assets/22287261/0a8d7f6f-6ff3-402a-923f-9d968e4457d4">
